docs/user-guide/global-configurations/chart-repo.md

```diff
@@ -0,0 +1,56 @@
+# Chart Repository
+
+The Global configuration provides a feature of `chart repository` in which you can add your chart repository here.
+
+## Add Chart Repository
+
+Select the Chart Repository section of global configuration and click on `Add Repository` to add new chart, which provide three inputs as below.
+
+1. Name
+2. URL
+3. Authentication type
+
+![](../../.gitbook/assets/gc-add-chart.png)
+
+### 1. Name
+
+Provide a `Name` to your Chart Repository. It will be added as one option in the Chart Repository drop-down inside the Chart Repository section.
```

Comment:
User will not be able to understand as this required deep product knowledege... it should be something like - 'This name is added as prefix to the name of the chart in the listing on the helm chart section of application' and add image to show it